content/derisking-government-tech/home/buying-development-services.md

```diff
@@ -156,7 +156,7 @@ Proposals are evaluated as follows:
 
 Inform vendors their proposed technical approach will be evaluated for how it describes its approach to modern software development practices. In particular, ask them to include the process they will follow to meet the solicitation’s quality expectations for the software. Ask them also to identify any risks they anticipate in regard to the specific project’s development effort and how it would address them. 
 
-##### Staffing approach
+##### Staffing approach {#staffing-approach}
```

Comment:
clarification: Are heading anchor links not being auto-generated? Because that should be happening in the 11ty config setup.

Reply:
They are being auto-generated. I've manually updated a few of them to remove special characters to simplify links to them from elsewhere in the guide; in this case there are links elsewhere to this section, so I thought it would help to explicitly set the default heading anchor so that any future changes to the prose won't break the link.

If that goes against the usual pattern I'll change it for consistency's sake.